Documents/Coding Standards/CodingStandards.tex

```diff
@@ -34,5 +34,33 @@
   \subsection{Swift}
   For Swift we will use \href{https://swift.org/documentation/api-design-guidelines/}{Apple design guidelines}.
   
+  \section{Git Structure}
+  \subsection{Branching}
+  Rule one, always, is never work out of \texttt{master}. A new branch is created when someone wants to implement, or experiment with something that is not theirs, or in the \texttt{master}, so they will branch off of the relevant branch, to make their proposed changes.
+  \subsubsection{Naming}
+  A branch shall never contain an individuals name, rather it must be informative as to what the intention of the branch is, however, it is in the best interests of readability to keep the name short. An example of a terrible name is \texttt{Joe}; another is \texttt{JoeWebsiteChanges}, where the name contains a member name, and gives very little information as to what is going on in the branch. Bad, but barely acceptable names are \texttt{WebsiteChanges}, or \texttt{WebsiteExperimentation}. Ideal names are as follows: in the case of a branch where the website is ultimately being assembled, by merging other branches into it, and the only modifications taking place are to structure the files, or a quick fix, such as changing the colour of a button, \texttt{Website}; in the case that a new homepage is created for the website, where the homepage is created, and merged into \texttt{Website} on completion, \texttt{Website-Homepage}; finally, if infamous \textit{Joe}, wants to propose a change to the \texttt{Website-Homepage} branch, where he wants to change the colour of a button, but Joe has not been assigned, or is not the primary benefactor to \texttt{Website-Homepage}, he creates a new \texttt{Website-Homepage-ButtonColourChange} branch. Notice that in the ideal examples it is possible to track down the source of a branch, to understand exactly what it's purpose is.
   
-\end{document}
\ No newline at end of file
+  \subsection{Merging}
+  After changes have been made on the branch, the creator, who wants to merge, shall then make a pull request on GitHub, and announce the request on Slack, where it shall become the responsibility of the entire team, but more importantly that of the original branch maintainer to analyze and comment on the changes. After a sufficient amount of time and discussion has passed, either the branch maintainer (if it is a minor change), or the entire team (if it is to master, or a major change) shall decide to merge or not.
```

Comment:
I'm not a fan of using slack to discuss reviews. I really think we should be using GitHub reviewers and this review style to handle reviews as in this case we can make review comments inline which is much easier. Maybe slack just to notify the code owner that you have a PR.